torch/_dynamo/eval_frame.py

```diff
@@ -665,9 +664,10 @@ def graph_with_interpreter(*args):
     ).transform()
 
     # Make dynamo graph to have same input/output spec as user code
+    input_strs = [f"orig_arg_{i}" for i in range(len(args))] + list(kwargs.keys())
```

Comment:
Should this be conditional check where we only append kwargs list if it is not empty?

Reply:
@tugsbayasgalan I will merge this PR, but I will come back to discuss your concerns and hopefully address the corner case detected internally